src/components/RelatedVideoItem/RelatedVideoItem.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Image, GridItem, Title } from './RelatedVideoItem.styles';
+
+const RelatedVideoItem = ({ video, videos }) => {
+  const history = useHistory();
+  const { snippet } = video;
+
+  const onClickHandler = () => {
+    history.push({
+      pathname: 'watch',
+      search: `?id=${video.id.videoId}`,
+      state: { title: snippet.title, description: snippet.description, videos },
```

Comment:
Following previous idea:
```suggestion
      state: { title, description, videos },
```